packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/src/Providers/CollectionResultDefinition.cs

```diff
@@ -150,6 +170,12 @@ protected override ConstructorProvider[] BuildConstructors()
                 "client",
                 $"The {_client.Type.Name} client used to send requests.",
                 _client.Type);
+
+            // For next link operations, exclude the nextPage parameter from constructor
+            var constructorParameters = _paging.NextLink != null
```

Comment:
This logic can be deleted because the create request method no longer has the next page param.